api/app/signals/apps/xperimental/migrations/0001_initial.py

```diff
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+# Generated by Django 3.2.5 on 2021-12-01 11:51
+from django.db import migrations, models
+
+signal_filter_view_reverse_sql = 'DROP VIEW IF EXISTS "signals_filter_view";'
+
+signal_filter_view_sql = f"""
+{signal_filter_view_reverse_sql}
+CREATE VIEW "signals_filter_view" AS
```

Comment:
Not sure what you really mean here. But I added the "AS" to all joins. Just to keep it consistent in the query.
This query is partially based on the existing history view but it works a bit different so that's why there are several different JOINS like "LEFT" and "INNER".